mergify_engine/tests/functional/actions/test_edit.py

```diff
@@ -14,12 +14,18 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+import pytest
 import yaml
 
+from mergify_engine import config
 from mergify_engine.tests.functional import base
 
 
 class TestEditAction(base.FunctionalTestBase):
+    @pytest.mark.skipif(
+        not config.GITHUB_URL.startswith("https://github.com"),
+        reason="requires GHES 3.2",
+    )
```

Comment:
This means we won't test it against GHES >= 3.2 either? 😢

Reply:
For now yes, the integration run 3.1 so we are OKish, I created a ticket to retrieve the server version and be able to write a better skipif.

Reply:
would it be more understandable with `requires GHES==3.2` ? Otherwise LGTM

Reply:
It's actually >= 3.2